bevy_lint/docs/release.md

```diff
@@ -0,0 +1,51 @@
+# Release Checklist
+
+## Kick-off Pull Request
+
+1. Review the [changelog](../CHANGELOG.md) and ensure that all notable changes have been documented.
+2. Replace `[Unreleased]` heading with the version with the format `[X.Y.Z] - YYYY-MM-DD`.
+3. Update the `**All Changes**` link to compare from `main` to the new tag `lint-vX.Y.Z`. (E.g. `lint-v0.1.0...main` to `lint-v0.1.0...lint-v0.2.0`.)
+4. Remove the `-dev` suffix from the version in [`Cargo.toml`](../Cargo.toml), and ensure [`Cargo.lock`](../../Cargo.lock) also updates.
+5. Commit all of these changes and open a pull request.
+6. Merge the PR once a core Bevy maintainer approves it with no outstanding issues from other contributors.
+      - This starts the release process, enacting a freeze on all other changes until the release has finished.
+
+## Release on Github
+
+1. [Create a new Github release](https://github.com/TheBevyFlock/bevy_cli/releases/new).
+2. Set the tag to `lint-vX.Y.Z`.
+3. Set the title to `` `bevy_lint` - vX.Y.Z``
+4. Paste the following into the release description:
+
+````markdown
+One-sentence summary of changes. What awesome features can we spotlight? What critical bugs were fixed?
+
+This release uses the `nightly-YYYY-MM-DD` toolchain, based on Rust 1.XX.Y. You can install it from Git with the following commands:
```

Comment:
How do we determine what toolchain and Rust version to put here? Is it just the one from `rust-toolchain.toml` and the MSRV?